### PR TITLE
Add FreeTube v0.7.3

### PR DESCRIPTION
**Pull request by bgrnwd**
FreeTube is an open source desktop YouTube player built with privacy in
mind

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/freetube.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'freetube' do
+  version '0.7.3'
+  sha256 :no_check
```
Comment (miccal) [inline Casks/freetube.rb:3]: [Please add the `sha256` value.](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md)

**Review by ran-dall — CHANGES_REQUESTED** [review]
@bgrnwd There's a couple things missing.

File: Casks/freetube.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+  url "https://github.com/FreeTubeApp/FreeTube/releases/download/v#{version}-beta/FreeTube-#{version}-mac.dmg"
+  appcast 'https://github.com/FreeTubeApp/FreeTube/releases.atom'
+  name 'FreeTube'
+  homepage 'https://github.com/FreeTubeApp/FreeTube'
+
+  app 'FreeTube.app'
+end
```
Comment (ran-dall) [inline Casks/freetube.rb:11]: You might be missing an `zap` as well.

File: Casks/freetube.rb (hunk 1 above)
Comment (bgrnwd) [reply Casks/freetube.rb:3]: @miccal Hey sorry, first time submitting a PR to `homebrew`, are you referring to the `sha256` of `FreeTube.app` or `freetube.rb`? 

FYI the application is not currently [signed](https://github.com/FreeTubeApp/FreeTube/wiki/F.A.Q.#my-virus-protection-gave-a-warning-when-i-tried-to-run-freetube--is-freetube-a-malicious-application)

Comment (miccal) [reply Casks/freetube.rb:3]: @bgrnwd the `sha256` of the downloaded file `FreeTube-0.7.3-mac.dmg`.

File: Casks/freetube.rb (hunk 2 above)
Comment (bgrnwd) [reply Casks/freetube.rb:11]: If the only file that is installed is `/Application/FreeTube.app` is `zap` required?

Comment (ran-dall) [reply Casks/freetube.rb:11]: @bgrnwd Probably. Most of the time app have support files and what not. You can use @nrlquaker's [createzap tap](https://github.com/nrlquaker/homebrew-createzap) or you can use use an our developers tools to track down exactly which files those are or even use an uninstaller (such as [AppCleaner](https://freemacsoft.net/appcleaner/)) to find all the files..

**Comment by ran-dall** [comment]
@bgrnwd FWIW you can use `brew cask style --fix <cask>` it automagically fix the `style`

**Review by ran-dall — APPROVED** [review]

**Pull request merged**